### ISSUE TITLE
Fix Travis builds

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-installer#243]
The Travis builds for silverstripe-installer have been broken for some time. We need to fix them.

PRs:
 - [x] #244

**Comment by chillu** [comment]
Thanks for highlighting that. I know @dnsl48 is going to be all over this :D

**Comment by dnsl48** [comment]
It looks like I accidentally fixed this when was testing my other stuff locally...

**Comment by maxime-rainville** [comment]
If it's been accidentally fix, can we accidentally close the issue?

**Comment by dnsl48** [comment]
Well, I guess after an accidental merge we could do that :D

**Comment by robbieaverill** [comment]
![image](https://user-images.githubusercontent.com/5170590/50756753-7d99ca00-125d-11e9-83b9-d56f647efd21.png)

I'm still being blamed, so let's not close this issue yet 😛

**Comment by dnsl48** [comment]
> Ok, I've done some investigation and it looks like currently PHPUnit tests on silverstripe/installer are false-green, as they are actually don't run any tests at all, because of --prefer-dist.

> The first PR (#244) switches to --prefer-source, which fixes the behat tests, but also makes Travis to run the PHPUnit tests which it hasn't been running so far.

> Thus, one of the jobs runs out of time and another has a test broken.

Even after the merge we cant close this issue. We'll need to do more work to fix the PHPUnit test issues.

**Comment by dnsl48** [comment]
Still need to fix the tests.
This time it's `silverstripe/postgresql` version in .travis.yml

**Comment by robbieaverill** [comment]
I’m happy to do it out of hours, I’d rather do that than close the issue while the builds are still red (personally)

**Comment by dnsl48** [comment]
@robbieaverill we're still keeping an eye on the red builds, but at this point it's rather a number of issues spread across different repos.
E.g. `4.1` and `4.3` are fixed already (even though 4.1 tests sometimes take longer than 50 minutes to pass, which errors on TravisCI) . `4.2` and `4` have some issues, but those are a number of different things which are spread across different repositories. I think there's no real value in a separate ticket for all of them, as we'll have to approach them separately anyways.

**Comment by robbieaverill** [comment]
Cool, fair enough

**Comment by chillu** [comment]
Yeah it's more of a point about team workflow and responsibilities. I'm very keen to get the builds fixed, but I don't think creating an issue about it is the way to go. Broken builds should be in-your-face, and we've got that now with the dashboard in the Wellington team area. I've asked @maxime-rainville to work out a way to get that going in Auckland in a visible+obvious way as well.

**Comment by maxime-rainville** [comment]
The URL I had didn't have the right IP. Guy gave me the right one.
